docs/notebooks/notes/gim.py

```diff
@@ -0,0 +1,166 @@
+import numpy as np
```

Comment:
generatung_image_masks:

- I have added some notations but since the whole codes are in one iteration, I couldn't add description markdown cell 

- I have updated visualize.py in /brainlit/viz

brainlit_ffn:

- I have added some more descriptions. The details of the most steps in section B can be found in section C

image_processing.py & benchmarking_parameters.py:

- updated